### DISCUSSION TITLE
Minor correction to README.md

### DISCUSSION THREAD

**Discussion by ArchieLinux** [BurntSushi/ripgrep#2380] [General]
Tried to submit a PR with a corrected README.md, but wasn't accepted (probably new user error on my part with Git). Edit proposed for Arch Linux installation section: 

CURRENT: ` $ pacman -S ripgrep`

PROPOSED: `$ sudo pacman -S ripgrep`

(root access rights are needed to install)

Hope this clarification is useful.

**Comment by BurntSushi** [comment]
I don't think there is any one accepted way to present this. The README is actually seemingly inconsistent with respect to whether it includes `sudo` in the install command or not. Presumably, `sudo` is something that the _user_ should know to type when it's necessary. It isn't _always_ necessary, for example, if you're already running as root. (As you might during the initial installation of Archlinux.)

**Reply by ArchieLinux** [reply]
Thanks for your reply and the great coding., FWIW, the convention (when root is required) would be` # pacman -S ripgrep` rather than `$ pacman -S ripgrep` - the distinction is to help newbies (me?) not get stuck with commands that don't work ;-)
Cheers. (Still not sure why I was unable to submit a PR having used Github CLI to authenticate, etc.)
